packaging/hudi-utilities-bundle/pom.xml

```diff
@@ -172,6 +172,10 @@
                   <pattern> org.apache.htrace.</pattern>
                   <shadedPattern>org.apache.hudi.org.apache.htrace.</shadedPattern>
                 </relocation>
+                <relocation>
+                  <pattern>org.eclipse.jetty.</pattern>
+                  <shadedPattern>org.apache.hudi.org.apache.jetty.</shadedPattern>
```

Comment:
@pratyakshsharma is the target/shaded pattern right?  we did for the spark bundle. can we just follow suit?

Reply:
@vinothchandar Yes this is how it was done for spark-bundle as well. Let me re-trigger the build for this.